src/main/resources/freemarker/researcher-dar-approved.html

```diff
@@ -0,0 +1,77 @@
+                <p align="left" style="font-family: 'Roboto', sans-serif; margin-top: 0;">${dataUseRestriction}</p>
+                <hr style="border: none; border-bottom: 1px solid #cccccc;">
+                <p style="margin: 20px 0 0 0;">Please remember you have attested to the following in your data access request:</p>
+            </td>
+        </tr>
+        <tr width="600" style="font-family: 'Roboto', sans-serif ;">
+            <td align="left" style="border-collapse: collapse; font-family: 'Roboto', sans-serif ; font-size: 16px; color: #777777; text-align: left; line-height: 25px; padding: 0px 30px 0 30px;">
+                <ol style="margin: 5px 0 5px 0; font-size: 15px;">
+                    <li>Data will only be used for approved research.</li>
+                    <li>Data confidentiality will be protected and the investigator will never make any attempt at “re-identification”.</li>
+                    <li>All applicable laws, local institutional policies, and terms and procedures specific to the study’s data access policy will be followed.</li>
+                    <li>No attempts will be made to identify individual study participants from whom data were obtained.</li>
+                    <li>Data will not be sold or shared with third parties.</li>
+                    <li>The contributing investigator(s) who conducted the original study and the funding organizations involved in supporting the original study will be acknowledged in publications resulting from the analysis of those data.</li>
+                </ol>
+            </td>
+        </tr>
+        <tr width="600" style="font-family: 'Roboto', sans-serif ;">
+            <td align="left" style="font-family: 'Roboto', sans-serif ; font-size: 16px; color: #777777; text-align: left; line-height: 25px; padding: 0 30px 10px 30px;">
+                <p>You can review your approved applications and attestation statement via the DUOS researcher console.</p>
+                <p>You should be receiving invitations to access these datasets via the FireCloud platform within the next 24 hours (access was designated to <a style="text-decoration:none!important; color:#777777!important;">${researcherEmail}</a>)</p>
+                <p>Please reach out if you have any questions or concerns.</p>
+                <p>Thank you,<br>the DUOS team</p>
+            </td>
+        </tr>
+    </table>
+    <table width ="600" style="font-family: 'Roboto', sans-serif ; margin-top: 15px;">
+        <tr align="center" style="font-family: 'Roboto', sans-serif ; color: #999999;">
+            <td align="center" valign="middle" style="padding: 0 10px 0 10px; border-collapse: collapse; font-family: 'Roboto', sans-serif; font-size: 14px; color: #999999; text-align: center; line-height: 21px; display: inline-block; vertical-align: middle;">	&#169; Broad Institute </td>
+        </tr>
+    </table>
+</center>
```

Comment:
`center` is a deprecated tag - not sure how this works in email. For PRs like this, it's also helpful to have screenshots of the emails that go out. It makes it a lot easier to review the PR.

Reply:
Thank you @nlopezzalba ! Disregard my `center` comment then - makes sense that you need it.

Reply:
sure, thank you!